Missions/Missions/Games Tab/BalanceGameViewController.swift

```diff
@@ -120,7 +143,7 @@ class BalanceGameViewController: UIViewController {
     
     private lazy var rightBall: UIView = {
         let rightBall = UIView(frame: .zero)
-        rightBall.backgroundColor = UIColor(red:1.00, green:0.34, blue:0.34, alpha:1.0)
+        rightBall.backgroundColor = UIColor(red:1.00, green:0.34, blue:0.34, alpha:0.0)
```

Comment:
Why are the alpha's `0.0` here instead of `1.0`?

Reply:
I made this alpha 0.0 because I'm going to delete it at some point, because I created the cross hairs in GameScene. 

However, I only made single ball game for now. I want to have the game logic tested before making two ball game, since two ball game is almost a mirror of single ball.